rocAL/rocAL_pybind/examples/rocAL_api_coco_pipeline.py

```diff
@@ -18,20 +18,16 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
```

Comment:
For the anchors, it is better to load from a binary file instead of txt, so the size will be less